samples/apps/basic/basic.cpp

```diff
@@ -322,6 +331,12 @@ namespace basicapp
           }
           // End of Authorization Check
 
+          // Make operation auditable by writing user-supplied COSE sign1
+          // document to the ledger
+          auto audit = ctx.tx.template rw<AuditValue>(
+            fmt::format("{}.cose_audit", CUSTOM_ENDPOINTS_NAMESPACE));
+          audit->put(ctx.rpc_ctx->get_request_body());
```

Comment:
Does this body include a claim of what the operation is (`custom_endpoints` vs `runtime_options`)? I don't see any COSE protected header here, but think that we should make it explicit. With other auth schemes, where an app may only write a UID/OID for the submitter, I think it should still make clear which operation is recorded.

Reply:
I am doing separately in a replay protection PR, but yes that is needed, and that will mean new authn methods.